Toolkit/ArcGISToolkit/AR/ArcGISARView.swift

```diff
@@ -330,7 +331,6 @@ public class ArcGISARView: UIView {
     fileprivate func didStartOrFailWithError(_ error: Error?) {
```

Comment:
if this takes a `ArcGISARView.CoreLocationError` instead of an `Error`, then calling code can/should just look like this:

```swift
didStartOrFailWithError(.missingPListKey)
```

Reply:
It takes an `Error`, so `CoreLocationError.missingPListKey` works.